CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -128,6 +136,14 @@ public static function add(&$params) {
       $financialAccount->opening_balance = $financialAccount->current_period_opening_balance = '0.00';
     }
     $financialAccount->save();
+
+    // invoke post hook
+    $op = 'create';
+    if (!empty($params['id'])) {
+      $op = 'edit';
+    }
```

Comment:
@pradpnayak isn't this a repetition? I mean can we not initialise the $op parameter once ? and need optimisation 
```$op = empty($params['id']) ? 'create' : 'edit';```